R/WebApi.R

```diff
@@ -142,3 +142,39 @@ getCdmSources <- function(baseUrl) {
 
   do.call(rbind, sourceDetails)
 }
+
+
+#' Get the WebApi configuration for the WebApi instance.
+#'
+#' @details
+#' Get the WebApi configuration for the WebApi instance.
+#'
+#' @param baseUrl        The base URL for the WebApi instance, for example:
+#'                       "http://server.org:80/WebAPI".
+#' @param sourceKeys     A list of string names for the source keys as in
+#'                       configuration page of Atlas.
+#' @return
+#' A list representing the WebApi configuration for the CDM Schema
+#'
+#' @examples
+#' \dontrun{
+#' #
+#'
+#' getConfiguration(baseUrl = "http://server.org:80/WebAPI", sourceKeys = c('source1', 'source2'))
+#' }
+#'
+#' @export
+getConfiguration <- function(baseUrl,sourceKeys){
```

Comment:
This function name is very vague. I literally have no idea what it will return. Your documentation says the function will return 'A list representing the WebApi configuration for the CDM Schema'. Could you be more specific? What would I need this for? Maybe in the example show how the result would be used, for example printing the list of CDM sources (if that is what it returns).